packages/api/tests/endpoints/test-granules.js

```diff
@@ -353,22 +360,28 @@ test.beforeEach(async (t) => {
   const QueueName = randomString();
   const { QueueUrl } = await sqs().createQueue({ QueueName }).promise();
   t.context.QueueUrl = QueueUrl;
-  const getQueueAttributesResponse = await sqs().getQueueAttributes({
```

Comment:
Nit: readability

Reply:
https://github.com/nasa/cumulus/pull/3225/commits/ea12a2846db17b78e9107a5109a445018d7ddfc8

Reply:
👍🏻  Thank you!